tests/conftest.py

```diff
@@ -85,12 +85,101 @@ def docker_alluxio():
     stop_docker(master_container)
 
 
+@pytest.fixture(scope="module")
+def docker_alluxio_with_etcd():
```

Comment:
Same here @ssyssy The core requirement of coding is to minimize duplicate code as much as possible.
Please use smaller methods to avoid duplicate codes